src/Button/DigitizeButton/DigitizeButton.jsx

```diff
@@ -552,12 +552,20 @@ class DigitizeButton extends React.Component {
       drawType,
       map,
       onDrawEnd,
-      onDrawStart
+      onDrawStart,
+      digitizeLayerName
     } = this.props;
 
     let geometryFunction;
     let type = drawType;
+    let digitizeLayer = MapUtil.getLayerByName(map, digitizeLayerName);
 
+    // this checks to see if the digitizeLayer  is in the map 
```

Comment:
too much whitespace

Reply:
it will take awhile until I get the hang of this whitespace rules :D